server/routerlicious/packages/services-shared/src/healthCheckEndpoints.ts

```diff
@@ -80,7 +74,6 @@ export function createHealthCheckEndpoints(
 				readinessProbeMetric.error("Readiness probe failed");
 				response.sendStatus(503);
 			} else {
-				readinessProbeMetric.success("Readiness probe successful");
 				response.sendStatus(200);
 			}
 		},
```

Comment:
@zhangxin511 mentioned there is possibility for unclear behavior here if `isReady()` throws an error. To address, consider just adding `.catch(() => false)` or even `.catch((error) => { Lumberjack.error("Error in ReadinessChecker", error); return false; })`

Reply:
Brainstorming:

```ts
const isReady = (await readinessCheck?.isReady().catch((error) => {
  Lumberjack.error("Unexpected exception in readiness check", probeProps, error);
  return false;
})) === true;
if (isReady) {
  response.sendStatus(200);
} else {
  response.sendStatus(503);
}
```

Reply:
I think I might modify the returned object to include an exception object. Something like:

```
export interface IReadinessStatus {
	/**
	 * Whether the service/functionality is ready for use.
	 */
	ready: boolean;

	/**
	 * Optional exception if an error occurs.
	 */
	exception?: any;
}
```

Then based on the response, we can log the exception as well and also return an HTTP status.